### PR TITLE
Version 3.0.2

### PR DESCRIPTION
**Pull request by MarcinWieczorek**
I haven't checked any things related to dependencies. Please look into it and let me update the branch. I hope that's just a bump tho.

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 pkgname=gitkraken
 pkgrel=2
```
Comment (KillWolfVlad) [inline PKGBUILD]: > pkgrel=2

Why?

Comment (MarcinWieczorek) [reply PKGBUILD]: Because I forget about it every single time.

Comment (Azd325) [reply PKGBUILD]: It would be good if you change it down to 1. :)

Comment (MarcinWieczorek) [reply PKGBUILD]: Sure I will, just got my PC on

**Comment by Azd325** [comment]
Thanks can you push to the AUR @MarcinWieczorek

**Pull request merged**